Update section about Custom Events Facebook Pixel (#4540)
segmentio/segment-docs: src/connections/destinations/catalog/facebook-pixel/index.md

### Custom Events

> To send *Custom* events, send any event that does not appear in either mapping. All properties you include in the event will be included as event properties. Any events you don't add to the "Map Your Events to Standard FB Events" setting will be sent to Facebook as a custom event. There is no setting to add custom events in the Facebook Pixel Destination configuration.

Reviewer: ```suggestion
To send *Custom* events, send any event that does not appear in either mapping. All properties you include in the event are included as event properties. Any events you don't add to the "Map Your Events to Standard FB Events" setting are sent to Facebook as a custom event. There is no setting to add custom events in the Facebook Pixel Destination configuration.
```
